packages/fscomponents/src/components/ZoomCarousel/types.ts

```diff
@@ -53,9 +53,39 @@ export interface ZoomCarouselProps {
   hidePageIndicator?: boolean;
 
   /**
-   * The styling used if fill container is set to true (that makes all carousel elememts expand)
+   * The styling of the container that holds the image carousel and thumbnails
    *
-   * @example {{height: '50%'}}
+   * @example {{flex: 1}}
    */
-  fillContainerStyle?: StyleProp<ViewStyle>;
+  carouselContainerStyle?: StyleProp<ViewStyle>;
```

Comment:
It doesn't look like this is used anywhere.

Reply:
https://github.com/brandingbrand/flagship/pull/1093/files#diff-badb4df4ea5cc3f25061a62fb1d31be9R362 

line 362

Reply:
disregard I cannot read